ansible_navigator/actions/config.py

```diff
@@ -211,7 +212,15 @@ def _run_runner(self) -> None:
                     return
                 ansible_config_path = exec_path
 
-            kwargs.update({"cmdline": self._args.cmdline})
+            if self._args.cmdline is not C.NOT_SET:
```

Comment:
just a note, importing the Constants could be avoided by checking if cmdline is instance list